app.rb

```diff
@@ -0,0 +1,199 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './book'
+
+# rubocop:disable Metrics/ClassLength
+# App class
+class App
+  attr_reader :book, :people, :rental
+
+  def initialize
+    @book = []
+    @people = []
+    @rental = []
+  end
+
+  # method to run App
+  def run
+    puts library_menu
+    input = gets.chomp
+    if input <= '6'
+      options(input)
+    elsif input == '7'
+      puts 'Thanks for using our App'
+    else
+      puts 'Please selecct a number from 1 - 7'
+    end
+  end
+
+  # library menu method
+  def library_menu
+    puts ''
+    puts ''
+    puts 'Welcome to School Library App!'
+    puts ''
+    puts ''
+    puts 'Please choose one option by entering a number'
+    puts ''
+    puts ''
+    puts '1 - List all  books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  # define options method
+  def options(input)
+    case input
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    else
+      'Enter a number btw 1 - 7'
+    end
+  end
+
+  # define show_menu method
+  def show_menu
+    puts ''
+    print 'Press Enter to go back to menu'
+    gets.chomp
+    puts ''
+    run
+  end
+
+  # define list_books method
+  def list_books
+    if @book.empty?
+      puts 'No books'
+      show_menu
+    end
+    puts 'List all books:'
+    puts ''
+    @book.each_with_index do |item, idx|
+      puts "#{idx} Title: #{item.title}, Author: #{item.author}"
+    end
+    show_menu
+  end
+
+  # define list_people method
+  def list_people
+    if @people.empty?
+      puts 'No people'
+      show_menu
+    end
+    puts 'List of people:'
+    puts ''
+    @people.each_with_index do |item, idx|
+      puts "#{idx} -- [#{item.class}] Name: #{item.name}, Age: #{item.age}"
```

Comment:
- [ ] Kindly display the Person's id as well just like in the design shown in the [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
![image](https://user-images.githubusercontent.com/46839548/209434498-52a81847-da53-4fc3-a6f3-1b5c924a345f.png)